src/string/CreditCard.ts

```diff
@@ -0,0 +1,168 @@
+/**
+ * TODO: Add module comment
+ *
+ * @since 0.0.3
+ */
+import { Kind, Kind2, URIS, URIS2, HKT2 } from 'fp-ts/HKT'
+import * as D from 'io-ts/Decoder'
+import * as Enc from 'io-ts/Encoder'
+import * as Eq_ from 'fp-ts/Eq'
+import * as G from 'io-ts/Guard'
+import * as Str from 'fp-ts/string'
+import * as TD from 'io-ts/TaskDecoder'
+import * as t from 'io-ts/Type'
+import { flow, pipe } from 'fp-ts/function'
+import * as fc from 'fast-check'
+import * as RA from 'fp-ts/ReadonlyArray'
+
+import * as Arb from '../internal/ArbitraryBase'
+import { MonoidSum } from 'fp-ts/lib/number'
+import { digits } from '../internal/util'
+
+/**
+ * @since 0.0.3
+ * @category Internal
+ */
+interface CreditCardBrand {
+  readonly CreditCard: unique symbol
+}
+
+/**
+ * TODO: Add module comment
```

Comment:
Here too -- I've been mostly just reusing the same comment from the top of the file.